gradle.properties

```diff
@@ -26,33 +26,33 @@ stdlibOsVersion=1.8.0
 stdlibFileVersion=1.9.0
 
 # Ballerinax Observer
-observeVersion=1.2.0
-observeInternalVersion=1.2.0
+observeVersion=1.2.3-20240408-194900-a0e331f
+observeInternalVersion=1.2.2-20240408-195500-1436319
 
 # Transitive Dependencies
 # Level 01
-stdlibConstraintVersion=1.4.0
+stdlibConstraintVersion=1.5.0
 stdlibUrlVersion=2.4.0
 
 # Level 02
-stdlibCryptoVersion=2.5.0
+stdlibCryptoVersion=2.7.0-20240410-115200-2ae9fd5
 stdlibTaskVersion=2.5.0
 
 # Level 03
-stdlibCacheVersion=3.7.0
+stdlibCacheVersion=3.8.0-20240409-101100-e2d7f4e
 stdlibMimeVersion=2.9.0
-stdlibUuidVersion=1.7.0
+stdlibUuidVersion=1.8.0-20240410-125400-bbe7ac5
 
 # Level 04
-stdlibAuthVersion=2.10.0
-stdlibJwtVersion=2.10.0
-stdlibOAuth2Version=2.10.0
+stdlibAuthVersion=2.11.0-20240410-121600-c514b62
+stdlibJwtVersion=2.11.0-20240410-124800-1a4eb44
+stdlibOAuth2Version=2.11.0-20240410-121700-502990d
 
 # Level 05
-stdlibHttpVersion=2.10.0
+stdlibHttpVersion=2.11.0-20240410-121300-ae1095d
 
 # Level 06
-stdlibTransactionVersion=1.8.0
+stdlibTransactionVersion=1.9.0-20240410-104900-57aec29
```

Comment:
Any specific reason for not using the docker image and using timestamp versions of library modules?

Reply:
Since the Snowflake connector is similar to other SQL connectors, I decided to use the same way to align with other SQL connectors. that is the only reason.

Reply:
I'd suggest using the connector builds for this since it is similar to the other connectors and maintaining the dependencies like this will be hard for a connector in the future. We have to bump the timestamp versions for each release and maintaining the dependent library versions manually might be cumbersome work. Given the automated workflow doesn't support ballerinax modules (this is the expected behavior since we use it for the modules packed with the distribution), we might miss this module to update to the latest version and even check for compatibility of the new distribution will be hard. 

I'd recommend using the connector builds for such connectors (except if there is an unavoidable reason such as having a compiler plugin such as NATS, RabbitMQ, Kafka, etc.) to make the processes easier.

Reply:
Create an issue to track https://github.com/ballerina-platform/ballerina-library/issues/6368